site/src/components/image-switcher/ImageSwitcher.module.css

```diff
@@ -0,0 +1,32 @@
+.container {
+  border: var(--salt-size-border) var(--salt-container-borderStyle) var(--salt-container-primary-borderColor);
+  padding: var(--salt-spacing-200);
+  gap: var(--salt-spacing-300);
+  display: flex;
+  flex-direction: column;
+  margin-block: var(--salt-spacing-300);
+}
+
+@media screen and (max-width: 996px) {
+  .container {
+    padding: var(--salt-spacing-100);
+    gap: var(--salt-spacing-200);
+  }
+}
+
+.container .switch {
+  --saltSwitch-height: var(--salt-switch-height);
```

Comment:
`--salt-switch-height` doesn't exist

Reply:
It's not defined here, this is to override a change in Mosaic